### PR TITLE
Started an admin payments dashboard

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@andrewconner @leogrim

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/views/admin/paymentsDashboard.scala.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@import com.keepit.controllers.admin.AdminPaymentsDashboard
+@(dashboard: AdminPaymentsDashboard)
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/views/admin/paymentsDashboard.scala.html:2]: I think I'm a fan of having a single-purpose `case class` for these admin pages. Makes it easier to read and understand, IMO.

File: kifi-backend/modules/shoebox/app/com/keepit/payments/AccountEventRepo.scala [hunk 2]
```diff
@@ -113,4 +109,7 @@ class AccountEventRepoImpl @Inject() (
     (for (row <- rows if row.accountId === accountId && (row.eventType === "user_added" || row.eventType === "user_removed")) yield row).sortBy(r => (r.eventTime asc, r.id asc)).list
   }
 
+  def adminGetRecentEvents(limit: Limit)(implicit session: RSession): Seq[AccountEvent] = {
+    rows.sortBy(ae => ae.eventTime desc).take(limit.value).list
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/payments/AccountEventRepo.scala:113]: We've fallen out of the habit of using `Limit` and `Offset`. I like them, though.

File: kifi-backend/modules/shoebox/app/views/admin/paymentsDashboard.scala.html (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/views/admin/paymentsDashboard.scala.html:2]: Nice, works for me.

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminPaymentsController.scala [hunk 3]
```diff
@@ -51,21 +51,17 @@ class AdminPaymentsController @Inject() (
         channel.push(s"Processing org ${org.id.get}: ${org.name}\n")
         db.readWrite { implicit session =>
           paidAccountRepo.maybeGetByOrgId(org.id.get) match {
-            case Some(_) => {
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminPaymentsController.scala:54]: Hah, a man after my own heart. I hate those extra `{}`s

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminPaymentsController.scala:54]: My other favorite languages include Python and Haskell. I think I might be a syntax snob.

**Comment by andrewconner** [comment]
Great start for the dashboard!

File: kifi-backend/modules/shoebox/app/views/admin/paymentsDashboard.scala.html (hunk 1 above)
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/views/admin/paymentsDashboard.scala.html:2]: :100:

**Pull request merged**